### PR TITLE
664 add delete to bobc

### LINKED ISSUE #664: `bobc`: add support for delete operation

### PR DESCRIPTION
**Pull request by lostl1ght**
Closes #664

### CODE REVIEW

**Review by idruzhitskiy — APPROVED** [review]

File: bob-apps/bin/bobc.rs [hunk 1]
```diff
@@ -437,6 +445,25 @@ async fn exist(keys: &KeyPattern, key_size: usize, client: &mut BobApiClient<Cha
     }
 }
 
+async fn delete(key: u64, key_size: usize, client: &mut BobApiClient<Channel>) {
+    let message = DeleteRequest {
+        key: Some(BlobKey {
+            key: get_key_value(key, key_size),
+        }),
+        options: Some(DeleteOptions::new_all()),
```
Comment (ikopylov) [inline bob-apps/bin/bobc.rs:453]: Once https://github.com/qoollo/bob/pull/621 has been merged, `None` will start working properly

**Review by ikopylov — APPROVED** [review]
One minor comment, otherwise LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ Bob versions changelog
 - Include bobc into release builds (#569)
 - Added blob-info and index-info features to brt (#356)
 - Support for files, file name patterns, key ranges and a 'exists' subcommand to 'bobc' (#539)
+- Add support for `delete` operation to `bobc` (#664)
```
Comment (ikopylov) [inline CHANGELOG.md:25]: This line should be in `[Unreleased]` section

**Review by ikopylov — APPROVED** [review]
LGTM

**Pull request merged**